monitor.go

```diff
@@ -25,33 +30,30 @@ func main() {
 	for {
 		event := <-listener
 
-		if event.Type == "container" && event.Action == "create" {
-			containerName := event.Actor.Attributes["name"]
-			log.Printf("New container (%s) created\n", containerName)
-
-			project, hasProject := event.Actor.Attributes["com.docker.compose.project"]
-			service, hasService := event.Actor.Attributes["com.docker.compose.service"]
-			oneoff := event.Actor.Attributes["com.docker.compose.oneoff"]
-
-			if hasProject && hasService {
-				config := docker.NetworkConnectionOptions{Container: event.Actor.ID}
-
-				if oneoff == "False" {
-					alias := service + "." + project + "." + domain
-					log.Printf("Attaching %s to the shared network with alias %s\n", containerName, alias)
-					config.EndpointConfig = &docker.EndpointConfig{
-						Aliases: []string{alias},
-					}
-				} else {
-					log.Printf("Attaching %s to the shared network\n", event.Actor.ID)
-				}
+		containerName := event.Actor.Attributes["name"]
+		log.Printf("New container (%s) created\n", containerName)
+
+		project, hasProject := event.Actor.Attributes["com.docker.compose.project"]
+		service, hasService := event.Actor.Attributes["com.docker.compose.service"]
+		oneoff := event.Actor.Attributes["com.docker.compose.oneoff"]
+
+		if hasProject && hasService {
+			config := docker.NetworkConnectionOptions{Container: event.Actor.ID}
 
-				err := cli.ConnectNetwork("shared", config)
-				if err != nil {
-					log.Fatal(err)
+			if oneoff == "False" {
+				alias := service + "." + project + "." + domain
+				log.Printf("Attaching %s to the shared network with alias %s\n", containerName, alias)
+				config.EndpointConfig = &docker.EndpointConfig{
+					Aliases: []string{alias},
 				}
+			} else {
+				log.Printf("Attaching %s to the shared network\n", event.Actor.ID)
+			}
+
+			err := cli.ConnectNetwork("shared", config)
```

Comment:
This sparked me the idea that we may have `dockerdev` also _create_ the shared network (instead of requiring the user to do so).

Reply:
That's likely possible, yes! We might even be able to customize the name (just like we can customize the domain for example).